asab/mom/amqp/subscription.py

```diff
@@ -14,7 +14,7 @@ def on_channel_open(channel):
 			channel.basic_qos(callback=on_qos_applied, prefetch_count=int(self.Broker.Config['prefetch_count']))
 
 		def on_qos_applied(method):
-			if parse_version(pika.__version__) >= parse_version('1.0.0'):
+			if parse_version(pika.__version__) > parse_version('0.13.1'):
```

Comment:
Nope, the correct test is `< 1.0.0`

Reply:
There are versions of pika `1.0.0b1` and `1.0.0b2` which are True for `< 1.0.0`. Maybe `< 1.a` ?

Reply:
yes